pubspec.yaml

```diff
@@ -46,6 +46,7 @@ dependencies:
   skeleton_text: ^3.0.0
   sliding_up_panel: ^2.0.0+1
   uni_links: ^0.5.1
+  web_socket_channel: ^2.2.0
```

Comment:
please remove this. ws 2.2.0 is a transitive dependency from v5.1.3 of graphql_flutter. 
no need to make it a main dependency
https://github.com/zino-hofmann/graphql-flutter/blob/main/packages/graphql/CHANGELOG.md